WebDriverAgentLib/Categories/XCUIElement+FBPickerWheel.m

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "XCUIElement+FBPickerWheel.h"
+
+#import "FBRunLoopSpinner.h"
+
+@implementation XCUIElement (FBPickerWheel)
+
+- (BOOL)fb_scrollWithOffset:(CGFloat)relativeHeightOffset
+{
+  NSString *previousValue = [self value];
+  XCUICoordinate *startCoord = [self coordinateWithNormalizedOffset:CGVectorMake(0.5, 0.5)];
+  XCUICoordinate *endCoord = [startCoord coordinateWithOffset:CGVectorMake(0.0, relativeHeightOffset * self.frame.size.height)];
```

Comment:
yep. I did not find any other good way of doing this. The PR itself was inspired by http://qualitytesting.tumblr.com/post/144152392209/moving-between-dynamic-picker-values-with-xctest
I've just added some stability improvements